plugin-server/src/capabilities.ts

```diff
@@ -9,13 +9,16 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
         case null:
             return {
                 ingestion: true,
+                ingestionOverflow: true,
                 pluginScheduledTasks: true,
                 processPluginJobs: true,
                 processAsyncHandlers: true,
                 ...sharedCapabilities,
             }
         case 'ingestion':
             return { ingestion: true, ...sharedCapabilities }
+        case 'ingestion-overflow':
+            return { ingestionOverflow: true, ...sharedCapabilities }
```

Comment:
I was reviewing my plan [here](https://github.com/PostHog/charts-clickhouse/pull/682/files#r1105808790) for the rollout and realised it won't work as I expected 😢 

Can you see a nicer way to roll out the new deployment? We could roll out with things colocated on ingestion to start with, then merge the deployment changes, what do you think?

Reply:
A bit of a chicken and egg situation. Wouldn't collocating things cause us to deploy multiple consumers in prod too? As these checks will be true for both `ingestion` and `ingestionOverflow` capabilities: https://github.com/PostHog/posthog/blob/master/plugin-server/src/main/ingestion-queues/kafka-queue.ts#L52

Reply:
Yeah. That will be an issue with the consumer group id being different 🤔 

Perhaps what we need to do is have an env var to disable the overflow producing functionality, then deploy both deployments, then enable the overflow functionality 😅